tests/foreman/cli/test_hostgroup.py

```diff
@@ -318,15 +316,16 @@ def test_positive_create_with_lifecycle_environment(self):
         :expectedresults: Hostgroup should be created and has lifecycle env
             assigned
 
-        :BZ: 1359694
+        :BZ: 1359694, 1313056
 
         :CaseImportance: Critical
         """
         org = make_org()
         lc_env = make_lifecycle_environment({'organization-id': org['id']})
         hostgroup = make_hostgroup({
             'lifecycle-environment': lc_env['name'],
-            'organization-id': org['id'],
+            'organization-ids': org['id'],
```

Comment:
as we have 'lifecycle-environment' by name it needs the organization to query/search that name.